src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax.model;
+
+public class FruitTransaction {
+    private String type;
```

Comment:
The field 'type' is declared but never used. Consider removing it if it's not needed, or ensure it's properly utilized if required by the application logic.